src/System Application/App/Filter Tokens/src/FilterTokensImpl.Codeunit.al

```diff
@@ -387,6 +387,7 @@ codeunit 58 "Filter Tokens Impl."
     local procedure ResolveTimeToken(var TimeFilter: Time; TimeToken: Text) Handled: Boolean
     begin
         TimeToken := DelChr(TimeToken); // Deletes all spaces in TimeToke
+        TimeToken := UpperCase(TimeToken); // Turn TimeToken into uppercase
```

Comment:
@JesperSchulz Well, it's Ok for the case statement, but I would probably prefer to pass the current value to the publisher (OnResolveTimeFilterToken).

I used lowercase in my blogpost in 2022 and this post had a lot of viewers :)) 

https://kepty.cz/2022/05/03/custom-filter-tokens/

Reply:
... and I see that we have also lowercase in two PTEs. We should probably convert the text to improve the stability, but this can break existing customizations.

Reply:
And it's also strange that ResolveDateFilterToken has the publisher before the standard code with the handled pattern, but OnResolveTimeFilterToken is as else statement. 

For the date procedure, the event is before the conversion to uppercase.

Reply:
OK, I'll fix the case only for now then.

Reply:
Now it merged the PR :-P I'll create a follow up to address this right away.